src/vellum/workflows/nodes/experimental/tool_calling_node/tests/test_node.py

```diff
@@ -127,3 +128,15 @@ class TestState(BaseState):
     assert isinstance(function_response.content, StringChatMessageContent)
     data = json.loads(function_response.content.value)
     assert data["generated_files"] == {"script.py": "print('hello world')"}
+
+
+def test_deployment_definition_release_tag_defaults_to_latest():
+    """
+    Test that when creating a DeploymentDefinition without specifying release_tag,
+    it defaults to "LATEST".
+    """
+    # WHEN we create a deployment definition without specifying release_tag
+    deployment_config = DeploymentDefinition(deployment="test-deployment")
+
+    # THEN the release_tag should default to "LATEST"
+    assert deployment_config.release_tag == "LATEST"
```

Comment:
[nit] we should update the annotation from `Optional[str] = "LATEST"` to `str = "LATEST"` since this model should never accept `None` as a value